src/openrct2/drawing/ScrollingText.cpp

```diff
@@ -34,7 +34,7 @@ struct rct_draw_scroll_text
     uint8_t bitmap[64 * 40];
 };
 
-constexpr int32_t MAX_SCROLLING_TEXT_ENTRIES = 32;
+constexpr int32_t MAX_SCROLLING_TEXT_ENTRIES = SPR_SCROLLING_TEXT_END - SPR_SCROLLING_TEXT_START;
```

Comment:
Normally, we would set this to a fixed value and define `SPR_SCROLLING_TEXT_END` as `SPR_SCROLLING_TEXT_START + 256`.

Reply:
I agree, however there is no header file specific for banners/scrolling text, Its a bit out of place to define it in sprites.h so I left it alone.